angular-portfolio/src/app/material/material.module.ts

```diff
@@ -0,0 +1,22 @@
+
+import { NgModule } from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+import { MatCardModule } from '@angular/material/card';
+import { MatTabsModule } from '@angular/material/tabs';
+import {MatToolbarModule} from '@angular/material/toolbar';
+import {MatButtonToggleModule} from '@angular/material/button-toggle';
+
+const Material = [
+  MatButtonModule,
+  MatCardModule,
+  MatTabsModule,
+  MatToolbarModule,
+  MatButtonToggleModule
+];
+
+@NgModule({
+  imports: [ Material ],
+  exports: [ Material ]
```

Comment:
The list of modules should not be exported. The only things that need to be exported or component classes that will be used in other modules. The actual components for the material components are exported within their respective modules already.

Reply:
What you're saying makes sense to me. However, the code does not function without the exports line. Is this because I made a new module for Materials rather than including it in the top level, and thus the components are not exported from the root?

Reply:
Yes, I think that's it. I didn't know you could do this, and it makes sense now that I look at it.